test/functional/sqlsrv/test_warning_errors2.phpt

```diff
@@ -15,7 +15,7 @@ if( $conn === false ) {
     die( print_r( sqlsrv_errors(), true ));
 }
 
-$stmt = sqlsrv_prepare( $conn, "SELECT * FROM [cd_info]");

```

Comment:
Were there only a few tests that used cd_info and test_streamable_types? The three or four which are modifed here can't be the only ones can they?

Reply:
You're right there are not many of those. Though there are other pre-populated tables used in sqlsrv tests, for pdo side there are less than a handful.